aws-api-appsync/src/main/java/com/amplifyframework/datastore/appsync/AppSyncExtensions.java

```diff
@@ -129,6 +129,15 @@ public String toString() {
             '}';
     }
 
+    /**
+     * Checks whether the current extension data indicates an authorization related error.
+     * @return True if it is authorization related, false otherwise.
+     */
+    public boolean isAuthorizationError() {
+        return AppSyncExtensions.AppSyncErrorType.UNAUTHORIZED.equals(this.getErrorType()) ||
+               AppSyncExtensions.AppSyncErrorType.UNAUTHORIZED_EXCEPTION.equals(this.getErrorType());
```

Comment:
Huh, I didn't realize AppSync returned both of these :/. That's kind of annoying.  I guess we should handle both though.